packages/arco-lib/src/components/Table/Table.tsx

```diff
@@ -196,6 +206,7 @@ export const Table = implementRuntimeComponent({
 
   useEffect(() => {
     mergeState({ pageSize });
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can we make sure the `pageSize` will never change or don't need to mergeState when it changes?

Reply:
This is only used to synchronise the initial state of pageSize